pystarport/pystarport/convert.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```

Comment:
could this be executed on the fly?

Reply:
protoc can be installed in nix
grpc plugin should be compiled,
i'll check  grpc plugin in nix